lib/phone_store/widgets/phone_card.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:flutter/material.dart';
+
+class CategoryStrings {
+  static const String smartphones = 'Smartphones';
+  static const String laptops = 'Laptops';
+  static const String videogames = 'Videogames';
+  static const String audio = 'Audio';
+  static const String appliances = 'Appliances';
+}
+
+class PhoneCard extends StatelessWidget {
+  final String image;
+  final String productName;
+  final String category;
+  final String stars;
+  final String discount;
+  final String price;
+  final double reviewScore;
+
+  const PhoneCard({
+    super.key,
+    required this.image,
+    required this.price,
+    required this.productName,
+    required this.category,
+    required this.discount,
+    required this.stars,
+    required this.reviewScore,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    double maxWidth = 149.413;
+
+    return FittedBox(
+      child: Card(
+        color: const Color(0xFFFFFFFF),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(12),
+          side: const BorderSide(
+            color: Color(0xFFE2E2E2),
+          ),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(12),
+          child: Row(
+            children: [
+              Container(
+                width: 79,
+                height: 79,
+                decoration: BoxDecoration(
+                  border: Border.all(
+                    color: const Color(0xFFF3F3F3),
+                    width: 1,
+                  ),
+                ),
+                child: ClipRRect(
+                  borderRadius: BorderRadius.circular(8),
+                  child: Image.asset(
+                    image,
+                    width: maxWidth,
+                    height: maxWidth,
+                    fit: BoxFit.cover,
+                  ),
+                ),
+              ),
```

Comment:
You should separate this widget as private widget and called _Image, to be easier to read code.